test/colony-network-mining.js

```diff
@@ -751,9 +740,9 @@ contract("ColonyNetworkMining", accounts => {
 
   describe("Rewards and punishments of good and bad submissions", () => {
     it("should punish all stakers if they misbehave (and report a bad hash)", async () => {
-      await giveUserCLNYTokensAndStake(colonyNetwork, MAIN_ACCOUNT, DEFAULT_STAKE);
       await giveUserCLNYTokensAndStake(colonyNetwork, OTHER_ACCOUNT, DEFAULT_STAKE);
       await giveUserCLNYTokensAndStake(colonyNetwork, OTHER_ACCOUNT2, DEFAULT_STAKE);
+      await advanceMiningCycleNoContest({ colonyNetwork, test: this });
```

Comment:
I can't work out why sometimes a new mining cycle is started in tests now and sometimes not?

Reply:
Basically anytime there is a `giveUserCLNYTokensAndStake` in the beginning of a test I added a `advanceMiningCycleNoContest` to "activate" that account's stake.